### PR TITLE
Notes api integration

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

File: src/context/notes-context.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  const [notes, setNotes] = useState([]);
+
+  const getNotes = async (token) => {
+    try {
+      const response = await axios.get("/api/notes", {
+        headers: { authorization: token },
+      });
+
+      setNotes(response.data.notes);
+      console.log(response);
```
Comment (Hamza-777) [inline src/context/notes-context.js]: Remove console logs if you don't need them anymore.

**Review by Hamza-777 — APPROVED** [review]
Everything else looks good, nice work.

File: src/context/notes-context.js (hunk 1 above)
Comment (abhisheksahani-nova) [reply src/context/notes-context.js]: Yupp thanks for pointing that out

**Review by VLeads — APPROVED** [review]
UI is clean. And code also looks good

**Pull request merged**